cmd/tetra/getevents/getevents.go

```diff
@@ -151,6 +161,19 @@ func New() *cobra.Command {
 				return fmt.Errorf("invalid value for %q flag: %s", "color", Options.Color)
 			}
 
+			var supportedEventTypes string
+
+			for _, v := range tetragon.EventType_name {
+				supportedEventTypes += v + ", "
+			}
+			supportedEventTypes = strings.TrimSuffix(supportedEventTypes, ", ")
+
+			for _, v := range Options.EventTypes {
+				if _, ok := tetragon.EventType_value[v]; !ok {
+					return fmt.Errorf("invalid value for %q flag: %s. Supported are %s", "event-types", v, supportedEventTypes)
+				}
+			}
```

Comment:
```suggestion
			for _, v := range Options.EventTypes {
				if _, found := tetragon.EventType_value[v]; !found {
					var supportedEventTypes string
					for _, v := range tetragon.EventType_name {
						supportedEventTypes += v + ", "
					}
					supportedEventTypes = strings.TrimSuffix(supportedEventTypes, ", ")
					return fmt.Errorf("invalid value for %q flag: %s. Supported are %s", "event-types", v, supportedEventTypes)
				}
			}
```

I would compute the error message only if needed:
```go
var supportedEventTypes string
for _, v := range tetragon.EventType_name {
	supportedEventTypes += v + ", "
}
supportedEventTypes = strings.TrimSuffix(supportedEventTypes, ", ")
```

Reply:
Makes sense. Thank you.